### PR TITLE
feat(DWS): add a new resource to manage snapshot policy

### PR DESCRIPTION
**Pull request by hmbig2**
**What this PR does / why we need it**:

**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```release-note

```

## PR Checklist

* [X] Tests added/passed.
* [X] Documentation updated.
* [X] Schema updated.

## Acceptance Steps Performed

```
Running tool: /usr/local/go/bin/go test -timeout 9000000000s -run ^TestAccDwsSnapshotPolicy_basic$ github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/dws -v

=== RUN   TestAccDwsSnapshotPolicy_basic
=== PAUSE TestAccDwsSnapshotPolicy_basic
=== CONT  TestAccDwsSnapshotPolicy_basic
--- PASS: TestAccDwsSnapshotPolicy_basic (1319.79s)
PASS
ok  	github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/dws	1319.834s
```

### CODE REVIEW

File: docs/resources/dws_snapshot_policy.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+```hcl
+  variable "cluster_id" {}
+
+  resource "huaweicloud_dws_snapshot_policy" "test" {
+    name       = "demo"
+    cluster_id = var.cluster_id
+    type       = "full"
+    strategy   = "0 8 6 4 * ?"
+  }
```
Comment (Lance52259) [inline docs/resources/dws_snapshot_policy.md]:
```suggestion
variable "cluster_id" {}

resource "huaweicloud_dws_snapshot_policy" "test" {
  name       = "demo"
  cluster_id = var.cluster_id
  type       = "full"
  strategy   = "0 8 6 4 * ?"
}
```

File: docs/resources/dws_snapshot_policy.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+    + **increment**: An incremental snapshot records the changes made after the previous snapshot was created.
+
+  Changing this parameter will create a new resource.
+
+* `strategy` - (Required, String, ForceNew) The strategy of the automated snapshot policy.  
+  Its format is a Cron expression, which specifies when to create a snapshot.
+
+  Changing this parameter will create a new resource.
+
+  -> Note: The UTC time is used by default. Set the policy based on the time zone and time difference as required.
```
Comment (Lance52259) [inline docs/resources/dws_snapshot_policy.md:77]: Is there any document for reference? such as [this one](https://support.huaweicloud.com/intl/en-us/usermanual-functiongraph/functiongraph_01_0908.html).

Comment (hmbig2) [reply docs/resources/dws_snapshot_policy.md:77]: NONE, it's linux cron expression.

File: docs/resources/dws_snapshot_policy.md (hunk 1 above)
Comment (hmbig2) [reply docs/resources/dws_snapshot_policy.md]: done

**Pull request merged**